### DISCUSSION TITLE
helper error

### DISCUSSION THREAD

**Discussion by pakuningratan** [leafsphp/leaf#221] [Q&A] [answered]
I got this error when use helper on my project:
**_Fatal error: Cannot declare class CustomHelper, because the name is already in use in G:\laragon\www\sipka\app\helpers\CustomHelper.php on line 3_**

**This my helper**
_\app\helpers\CustomHelper.php_

```
<?php 
class CustomHelper {
    public static function hello($text) {
        // perform date formatting operations
        return 'haloooooooooo '.$text;
    }
}
?>
```

**controller**
_app\controllers\jadwalController.php_

```
<?php
namespace App\Controllers;
 
use App\Helpers\CustomHelper;
 

class JadwalController extends Controller
{
    public function index()
    { 
        $text = 'bla bla bla bla';

        $halo = CustomHelper::hello($text);
        echo $halo;
    }
}
```

please give me suggestion, thanks

**Comment by mychidarko** [comment, marked as answer]
Hi @pakuningratan I'm not sure why it's saying your `CustomHelper` class already exists, maybe it's defined somewhere else in your code. However, I noticed that you left out the namespace in your helper. `namespace App\Helpers;`

**Reply by ibnsultan** [reply]
If the class is non existent it is then it's because the `namespace` is missing or the namespace does not follow the defined PSR standard i.e small case for the folders etc
